### PR TITLE
Adam add videos

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/writing-rules/data-flow/taint-mode.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ description: >-
 
 Semgrep supports intra-procedural [taint tracking](https://en.wikipedia.org/wiki/Taint_checking). This is a kind of data-flow analysis that tracks the flow of untrusted (aka "tainted") data throughout a program. The analysis produces a finding whenever such data goes into a vulnerable function (aka "sink"), without first having been checked or transformed accordingly (aka "sanitized").
 
+The following video can help you to understand taint mode:
+<iframe class="yt_embed" width="100%" height="432px" src="https://www.youtube.com/embed/6MxMhFPkZlU?start=25" frameborder="0" allowfullscreen></iframe>
+
```
Comment (adamkvitek) [inline docs/writing-rules/data-flow/taint-mode.md:14]: @efortuna I am tempted to move this to line 9 - so this video will become the actual introduction. We can let it start from the beginning also. :-)

File: docs/writing-rules/data-flow/taint-mode.md [hunk 2]
```diff
@@ -9,6 +9,9 @@ description: >-
 
 Semgrep supports intra-procedural [taint tracking](https://en.wikipedia.org/wiki/Taint_checking). This is a kind of data-flow analysis that tracks the flow of untrusted (aka "tainted") data throughout a program. The analysis produces a finding whenever such data goes into a vulnerable function (aka "sink"), without first having been checked or transformed accordingly (aka "sanitized").
 
+The following video can help you to understand taint mode:
```
Comment (efortuna) [inline docs/writing-rules/data-flow/taint-mode.md:12]:
```suggestion
The following video provides a quick overview of taint mode:
```

File: docs/writing-rules/data-flow/taint-mode.md (hunk 1 above)
Comment (efortuna) [reply docs/writing-rules/data-flow/taint-mode.md:14]: I think prooooobably start with the text. but yeah maaaaybe let the video start at the beginning as it gives a summary of what's to come in the video? not a strong preference though so I defer to you! starting at "25" jumps right to the goodies :-)

File: docs/writing-rules/data-flow/taint-mode.md (hunk 2 above)
Comment (efortuna) [reply docs/writing-rules/data-flow/taint-mode.md:12]: not a strong opinion on this one. could keep as you had originally written it.

**Review by efortuna — APPROVED** [review]
just little suggestions, but feel free to ignore all of them!

File: docs/writing-rules/data-flow/taint-mode.md (hunk 1 above)
Comment (adamkvitek) [reply docs/writing-rules/data-flow/taint-mode.md:14]: I like the start! Let's put it from the start. :-)

File: docs/experiments/taint-propagators.md [hunk 3]
```diff
@@ -8,6 +8,9 @@ description: "Taint propagators allow you to customize how taint is propagated."
 
 Taint propagators allow you to customize how taint is propagated. 
 
+The following video can help you to understand taint propagators:
```
Comment (adamkvitek) [inline docs/experiments/taint-propagators.md:11]:
```suggestion
The following video provides a quick overview of taint propagators:
```

**Pull request merged**